portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,28 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// Author: Jonathan Chavez - tabaresj@google.com
+var isTerminalOpen = false;
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+window.addEventListener("keydown", keyboardHandler);
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function keyboardHandler(e){
+  e = e || window.event;
+  console.log(String.fromCharCode(e.keyCode))
```

Comment:
Be careful with commiting console.logs, as far as I know, it shouldn't get to production

Reply:
Great, thanks for letting me know, I'll change that!